### DISCUSSION TITLE
Compile in 64 bit with the newest SDK for foobar 2.0

### DISCUSSION THREAD

**Discussion by aaronliu0130** [s0hv/foo_discord_rich#11] [Ideas]
Now that foobar 2.0 has released I think there should be a binary in 64 bit.

**Comment by s0hv** [comment]
I might try to do it at some point but for now it's on hold

**Reply by aaronliu0130** [reply]
Fritojay has an implementation at https://github.com/fritojay/foo_discord_rich

**Comment by s0hv** [comment]
I published my current work on 64bit support as a pre-release https://github.com/s0hv/foo_discord_rich/releases/tag/64-bit-prerelase

I only tested it very briefly so it might have some problems but hopefully it'll work without problems.
Also, it only supports foobar2000 version 2 and onwards as of now (hence why it's a pre-release).
If something does not work just report about it here.

**Reply by RJTimmerman** [reply]
Thank you for making this!
I am now using this version of foo_discord_rich, with the Catbox uploader for Powershell 7, and I had to increase the timeout limit to make it work consistently.
I now have another problem though: one of my albums creates the catbox url "https://files.catbox.moe/wgbxps.jpg;" (with a semicolon). I don't know why this happens (if it's to do with this component version, the uploader script, or catbox itself), but the program acts like nothing's wrong, while not displaying any image on Discord (just transparent to the profile background).

**Reply by s0hv** [reply]
If it's just this one album, you can manually set to url. Just select the tracks in the album, right click and select discrod rich presence and from there manually enter artwork url. Then you can remove the semicolon manually and set that as the new url
